lib/plug/debugger.ex

```diff
@@ -189,7 +222,9 @@ defmodule Plug.Debugger do
         session: session,
         params: params,
         style: style,
-        banner: banner
+        banner: banner,
+        actions: actions,
+        last_path: conn.request_path
```

Comment:
As I said on Slack, if it is a POST request, maybe the last_path should be the referer. If there is no referer, redirect to "/".